acceptance/setup/common/005_SyncTime.rb

```diff
@@ -8,6 +8,10 @@
 # Affects OSX and Ubuntu on vmpooler
 applicable_hosts = hosts.select{|host| host['platform'] !~ /osx|ubuntu/}
 
+# PCP-625 the beaker timesync method causes strange behavior on Windows Server 2016,
+# temporarily setting the date far into the past, which breaks SSL certificate retrieval
+applicable_hosts = hosts.select{|host| host['platform'] !~ /windows-2016/}
```

Comment:
Oh, this breaks skipping on ubuntu and osx. You need to modify `applicable_hosts`.